aioredis/util.py

```diff
@@ -55,6 +55,62 @@ def decode(obj, encoding):
     return obj
 
 
+class Queue:
+
+    def __init__(self, connection, key=uuid.uuid4().hex, max_size=0, timeout=None):
```

Comment:
Having such default value for key is **totally wrong**.
If someone using this Queue will not specify `key` parameter he will end up working with different queues every time application restarts.
key argument must not have default value.